resolwe_bio/processes/support_processors/multiqc.py

```diff
@@ -579,7 +579,10 @@ def run(self, inputs, outputs):
                     )
                 else:
                     count_report = "ReadsPerGene.out.tab.gz"
-                    report = f"{bam_name}.Log.final.out"
+                    if d.output.seqtk_downsampled == True:
```

Comment:
This will brake for legacy STAR data objects without the `seqtk_downsampled` output field